OrbitCore/CoreApp.h

```diff
@@ -45,11 +50,26 @@ class CoreApp {
                          const std::string& /*a_Module*/,
                          const std::string& /*a_Name*/) {}
   virtual void AddKeyAndString(uint64_t key, std::string_view str) {}
+  virtual bool SelectProcess(const std::string& a_Process) { return false; }
```

Comment:
This seems wrong. The only time it is used from GCoreApp and not GOrbitApp is in `Capture::LoadSession()`
Maybe consider the following option:
* Move `LoadSession` from Capture to OrbitApp, because it is only used in App.cpp and SessionsDataView and they both have access to GOrbitApp instead of just GCoreApp. 

If there is a good reason to have LoadSession in Capture, please leave it as is

Reply:
Good point, thanks, I made the change.